niimpy/EDA/EDA_countplot.py

```diff
@@ -149,38 +213,47 @@ def EDA_countplot(df, fig_title, plot_type = 'count', points = 'outliers', aggre
     assert ((isinstance(user,str)) or (user is None)), "user is not a string or None type."
     assert ((isinstance(column,str)) or (column is None)), "column in not a string or None type."
     
+    # Plot counts
     if plot_type == 'count':
         if aggregation == 'group':
-            n_events = df[['group', 'user']].groupby(['user', 'group']).size().to_frame()
-            n_events.columns = ['values']
-            n_events = n_events.reset_index()
-            EDA_boxplot_(n_events,
-                         fig_title,
-                         points, 
-                         y = 'values',
-                         xlabel="Group",
-                         ylabel="Count")
+                       
+            n_events = get_counts(df,aggregation)
+            print(n_events)
```

Comment:
Is it necessary to print something here? Or it was for debugging purposes?

Reply:
removed print